integr8ly/integreatly-operator
# New keycloak operator

Update to use the new [Keycloak Operator](https://github.com/keycloak/keycloak-operator). This required changes in the way realms, users and clients are handled:

in the 'old' keycloak operator clients and users were managed through the realm. When a user or client was updated, the realm was modified. This is no longer the case, users and clients are now their own CRs. Updates to realms are no longer reconciled.

Verifications steps:

1. Install all products using this branch and make sure single sign on still works as expected
1. Make sure user sso still works as expected
